### PR TITLE
Email and alerts edit email

### PR DESCRIPTION
**Pull request by leggsimon**
Adds the first part of the new Emails & Alerts page

## TODO:
- [x] Update icon to new pencil icon.
- [x] Accessibility on the `Edit` link!

## Desktop

screenshot 2017-03-03 14 47 30

## Mobile

screenshot 2017-03-03 14 47 40

cc/ @rebeccaturner17

### CODE REVIEW

File: client/components/alerts-v2/main.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
+@import "variables";
+
+.alerts-header {
+	-webkit-font-smoothing: antialiased;
+}
+
+.alerts--title {
+	@include oTypographySansBold(xl);
+	width: 100%;
+	margin-bottom: 0px;
```
Comment (debugwand) [inline client/components/alerts-v2/main.scss]: don't need px

File: client/components/alerts-v2/main.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+.alerts--description {
+	@include oTypographySans(m);
+	color: $sixty-percent-black;
+	margin-top: 5px;
+	width: 100%;
+}
+
+.alerts--email-subheading {
+	@include oTypographySansBold(m);
+	margin-bottom: 0px;
```
Comment (debugwand) [inline client/components/alerts-v2/main.scss]: no unit needed

File: client/components/alerts-v2/main.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+.alerts--email-description {
+	@include oTypographySans(m);
+	color: $sixty-percent-black;
+	margin-top: 5px;
+}
+
+.alerts--email-edit_link {
+	@include oTypographySansBold(m);
+	font-size: 18px;
```
Comment (debugwand) [inline client/components/alerts-v2/main.scss]: more a design question: are we *really* sure we want to start messing with the styles set out in https://github.com/Financial-Times/o-typography?

File: server/controllers/alerts-v2/read.js [hunk 4]
```diff
@@ -8,11 +9,16 @@ module.exports = (req, res) => {
...
-	res.render('alerts-v2/alerts', viewModel);
+	return getUserInformation(userId)
+		.then(data => {
+			viewModel.userInfo = data || {};
+		})
+		.then(() => {
+			res.set('Surrogate-Control', res.FT_NO_CACHE);
+			res.set('Cache-Control', res.FT_NO_CACHE);
+			res.render('alerts-v2/alerts', viewModel);
+		})
```
Comment (debugwand) [inline server/controllers/alerts-v2/read.js]: add catch

File: views/alerts-v2/alerts.html [hunk 5]
```diff
@@ -1,3 +1,14 @@
 <div class="mypage" id="site-content" role="main">
-	Hello, {{{userId}}}!
+	<div class="o-grid-container" data-trackable="alerts">
+		<header class="alerts-header">
+			<div class="o-grid-container">
+				<div class="o-grid-row">
+					<h2 class="alerts--title">Emails & Alerts</h2>
```
Comment (debugwand) [inline views/alerts-v2/alerts.html]: `&amp;` stops github etc flagging this

File: views/alerts-v2/alerts.html [hunk 6]
```diff
@@ -1,3 +1,14 @@
 <div class="mypage" id="site-content" role="main">
-	Hello, {{{userId}}}!
+	<div class="o-grid-container" data-trackable="alerts">
+		<header class="alerts-header">
+			<div class="o-grid-container">
+				<div class="o-grid-row">
```
Comment (debugwand) [inline views/alerts-v2/alerts.html]: wrap in `<div data-o-grid-colspan="12"></div>` for width settings.

File: views/alerts-v2/alerts.html [hunk 7]
```diff
@@ -1,3 +1,14 @@
...
+					<h2 class="alerts--title">Emails & Alerts</h2>
+					<p class="alerts--description">description</p>
+					<h3 class="alerts--email-subheading">Email Address</h3>
+					<p class="alerts--email-description">We'll send email newsletters to <strong>{{{userInfo.email}}}</strong><a href="//myaccount.ft.com/details/core/view?execution=e2s1" class="alerts--email-edit_link" aria-label="Edit your email address"><i class="alerts--email-edit_icon"></i>Edit</a></p>
```
Comment (debugwand) [inline views/alerts-v2/alerts.html]: `<strong>` can make screenreaders raise their voice to emphasise the word. span with a class will avoid that

Comment (laurieboyes) [inline views/alerts-v2/alerts.html]: Huh, interesting!

File: views/alerts-v2/alerts.html [hunk 8]
```diff
@@ -1,3 +1,14 @@
 <div class="mypage" id="site-content" role="main">
```
Comment (laurieboyes) [inline views/alerts-v2/alerts.html]: Re what I was saying about killing the `mypage` class, let's not worry about it right now. We seem to have it everywhere else across the app and it does seem to be depended on for one particular bit of lists-related styling 😖  https://github.com/Financial-Times/n-myft-ui/blob/c4afa3aa0465f4890e96f2f74b0fbc3fadb37537/myft/main.scss#L77

Sorry for trying to push you down a rabbit hole.

File: views/alerts-v2/alerts.html (hunk 7 above)
Comment (pixelandpage) [inline views/alerts-v2/alerts.html]: Very!

Comment (leggsimon) [inline views/alerts-v2/alerts.html]: Yeah, I didn't know that

File: server/controllers/alerts-v2/read.js (hunk 4 above)
Comment (laurieboyes) [inline server/controllers/alerts-v2/read.js]: I thought if you return a promise n-express it handles the rejection. No harm in the `catch(next)` below though.

**Pull request merged**